slemicro/combustion/09_podman.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+# Cockpit
```

Comment:
That's already part of the cockpit script. Do we need it here?

Reply:
My bad, copy & paste error